workflows/atacseq.nf

```diff
@@ -297,8 +297,29 @@ workflow ATACSEQ {
     //
     // SUBWORKFLOW: Filter BAM file
     //
+        // ch_bam_bai = PICARD_MARKDUPLICATES.out.bam
```

Comment:
Why this is commented? Might be that I don't see something since I am doing the review from my phone